devcontainers/templates/arch-linux/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# Distribution tag
+ARG TAG=base-devel
+
+FROM archlinux:${TAG} AS setup
+
+SHELL [ "/bin/bash", "-c"]
+
+RUN pacman -Syu --noconfirm     \
+    --disable-download-timeout  \
+    cmake python3 python-pip    \
+    xerces-c fox gdal proj      \
+    gl2ps jre17-openjdk swig    \
+    maven eigen flex bison git  \
+    gcc sdl2 libsm openmp       \
+    openscenegraph qt5-base     \
+    pacman-contrib sudo         \
+    boost crypto++ xorg         \
+    && pip install --break-system-packages conan \
```

Comment:
I think we need to install some font packages as well. The QtEnv GUI looks quite "strange" at the moment.